### DISCUSSION TITLE
Possibility to keep data when downgrading iOS version?

### DISCUSSION THREAD

**Discussion by ro1605he** [LukeZGD/Legacy-iOS-Kit#496] [Q&A] [answered]
@LukeZGD (or anyone who knows the answers to my questions!)  

I have an old iPhone 5 lying around that due to “storage full” no longer boots (Apple logo remains visible). I tried SSH Ramdisk, but have a feeling this won't work because of this either. Is it possible to use the script to downgrade from iOS (was running on 8.4.1 at the time of the crash) to iOS 7 without losing data? Or, when I choose option 1 (downgrade to 8.4.1) in the script, will it re-install this version without losing data?

Also, do you know if an iDevice gets low level formatted when doing a restore? Otherwise my last option could be to try a restore and recover deleted files afterwards?

**Comment by LukeZGD** [comment, marked as answer]
discussion of this was done in #498 for future reference
